routes/nodeMailer.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
```

Comment:
Let's keep all changes in this PR to the user and routes table to avoid overwriting the nodemailer files with the wrong changes when that PR gets merged in. Let's remove this file.